### DISCUSSION TITLE
Consistent Lap Spike Every Minute

### DISCUSSION THREAD

**Discussion by Zacoist** [ppy/osu#20956] [Q&A] [answered]
Hello everyone,
Problem: I've been experiencing an input lag spike (above +2) when it usually hovers below 2. I've timed the occurrence, and it happens every minute.

My CPU is good (i7-5820k, GTX 1070, 16 Gb RAM) with good storage on SSD. I've tried closing all my programs outside of osu. Default osu skin, Optimal/Unlimited, Compatibility mode, Reduced Frame Rate in osu settings, but the problem persists. I've even gone as far as to do a clean reinstall of my OS, but to no avail.

I noticed this problem about three days ago when I downloaded a windows update and the latest Nvidia driver. I returned to the previous Nvdia driver, but it didn't change anything. So I don't think it's the reason I was putting it here to show when I noticed the problem.

![screenshot012](https://user-images.githubusercontent.com/15700088/198120559-109f2a65-673c-46a4-871c-6010e53886a6.jpg)
![screenshot013](https://user-images.githubusercontent.com/15700088/198120573-9c1f91f7-775d-4fee-b2c9-6076b4d0267c.jpg)
![screenshot014](https://user-images.githubusercontent.com/15700088/198120586-ff6ad5e5-046b-4223-b466-974b92126c51.jpg)

[osu!auth.log](https://github.com/ppy/osu/files/9873233/osu.auth.log)
[runtime.log](https://github.com/ppy/osu/files/9873234/runtime.log)
[session.log](https://github.com/ppy/osu/files/9873235/session.log)
[update_success.log](https://github.com/ppy/osu/files/9873236/update_success.log)

If there is any possible solution or you need more information from me, let me know! I want to get back to playing osu!

**Comment by bdach** [comment, marked as answer]
Wrong version of the game. This repository is about lazer only.

Please ensure you've read https://osu.ppy.sh/wiki/en/Performance_troubleshooting in its entirety before proceeding.

**Comment by Zacoist** [comment]
New post found here: https://osu.ppy.sh/community/forums/topics/1666434?n=1
